src/reference/90-Developers-Guide/06-Datatype/00.md

```diff
@@ -5,12 +5,12 @@ out: Datatype.html
 Datatype
 --------
 
-sbt-datatype is a code generation library and an sbt autoplugin that generates
+`sbt-datatype` is a code generation library and an sbt autoplugin that generates
```

Comment:
Well, sort of yes and no. sbt-datatype in my head is a conceptual thing, like sbt is. It's not even a single artifact because it's made up of two parts: a library and a plugin: https://github.com/sbt/sbt-datatype/blob/master/build.sbt